src/main/archetype/ui.apps/src/main/content/META-INF/vault/filter.xml

```diff
@@ -3,6 +3,10 @@
     <filter root="/apps/${appId}/clientlibs"/>
     <filter root="/apps/${appId}/components"/>
     <filter root="/apps/${appId}/i18n"/>
+#if ( $includeForms == "y" || $includeFormsenrollment == "y" || $includeFormscommunications == "y" || $includeFormsheadless == "y")
+    <filter root="/apps/fd/af/themes"/>
+    <filter root="/apps/fd/af/theme-clientlibs"/>
+#end

```

Comment:
both filter should have conditional inclusion based on aem version.

Reply:
In future we do plan to add theme client libs incase a user migrates from 65 to cloud, didn't add explicit check for that

Reply:
Let's make it conditional for now. When we support it in the future, we will remove the condition.

Reply:
done